src/Sidenav/SideNavData.component.tsx

```diff
@@ -0,0 +1,97 @@
+import {
```

Comment:
This should just be a `SideNavData.ts` file since it only contains data (no JSX)